scannerlib/configchecks/file_check.go

```diff
@@ -53,13 +56,21 @@ type FileCheckBatch struct {
 }
 
 // Exec executes the file checks batched by the FileCheckBatch.
-func (b *FileCheckBatch) Exec() (ComplianceMap, error) {
+func (b *FileCheckBatch) Exec(prvRes string) (ComplianceMap, string, error) {
+
+	if b.filesToCheck.GetSingleFile() != nil{
```

Comment:
All modifications to FileSet is currently happening inside file_set.go (e.g. ApplyOptOutConfig, ApplyReplacementConfig). It'd be better if we could move this in there as a separate function.

Reply:
Done.